pkg/workload/workload.go

```diff
@@ -687,3 +689,8 @@ func AdmissionChecksForWorkload(log logr.Logger, wl *kueue.Workload, admissionCh
 	}
 	return acNames
 }
+
+func ReportEvictedWorkload(recorder record.EventRecorder, wl *kueue.Workload, cqName string, reason string, message string) {
```

Comment:
```suggestion
func ReportEvictedWorkload(recorder record.EventRecorder, wl *kueue.Workload, cqName, reason, message string) {
```

Nit. All are typed string.

Reply:
Fixed